packages/@uppy/image-editor/src/ImageEditor.tsx

```diff
@@ -77,14 +141,18 @@ export default class ImageEditor extends UIPlugin {
     return false
   }
 
-  save = () => {
-    const saveBlobCallback = (blob) => {
+  save = (): void => {
+    const saveBlobCallback = (blob: Blob | null) => {
       const { currentImage } = this.getPluginState()
 
+      if (!currentImage || !blob) {
+        throw new Error('currentImage and blob can not be null')
```

Comment:
This introduces a new runtime error – which would be a breaking change. Also you are not checking if it's null, making the error message confusing
```suggestion
      if (!currentImage) {
        throw new Error('Invalid plugin state: currentImage is not set')
```

Reply:
I think we should still fix the error message. If a end-user is getting `currentImage and blob can not be null`, there's no way they can understand what's going on.